app/models/card.rb

```diff
@@ -1,5 +1,16 @@
+
+VALID_EMOJIS = ["heart_eyes", "beer", "clap", "sparkling_heart", "heart_eyes_cat", "dog"]
+
 class Card < ApplicationRecord
   belongs_to :board
 
-  validates :title, presence: true, allow_blank: false
+  validate :valid_fields
+
+
+  def valid_fields
+    if ((self.text.nil? || self.text.length < 1) && \
```

Comment:
We can use a simpler check for the `text` field here:`if self.text.blank? && !VALID_EMOJIS